src/libraries/System.Net.Security/src/System/Net/Security/SslStream.Implementation.cs

```diff
@@ -411,7 +425,7 @@ private void SendBlob(byte[] incoming, int count)
 
                 if (_framing == Framing.Unified)
                 {
-                    _framing = DetectFraming(message.Payload, message.Payload.Length);
+                    _framing = DetectFraming(new Span<byte>(message.Payload, 0, message.Payload.Length));
```

Comment:
You defined DetectFraming to take a ReadOnlySpan... it'd be better then to construct a ReadOnlySpan rather than Span here.

Reply:
Also, it'd be cheaper to do `new ReadOnlySpan<byte>(message.Payload)` rather than `new ReadOnlySpan<byte>(message.Payload, 0, message.Payload.Length)`.

Reply:
Updated. Long term goal is to unify Sync and Async methods and use Adapters (or other methods) to distinguish IO.  So eventually, this whole method will go.